### PR TITLE
Fix CODEOWNERS file

### PR DESCRIPTION
**Pull request by kevinji**
I noticed that the file wasn't working as expected as all the usernames were missing the leading `@` sign.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -7,8 +7,8 @@
 #    https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
 # for more details.
 
-* alxiong philippecamacho sveitser
-/wallet/ jbearer shenkeyao
-/relayer/ nyospe
-/doc/ pictographer
-/.github/ Ancient123
+* @alxiong @philippecamacho @sveitser
```
Comment (sveitser) [inline .github/CODEOWNERS:10]:
```suggestion
* @philippecamacho @sveitser
```

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -7,8 +7,8 @@
...
+/wallet/ @jbearer @shenkeyao
+/relayer/ @nyospe
```
Comment (sveitser) [inline .github/CODEOWNERS:12]:
```suggestion
/eqs/ @nyospe
```

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -7,8 +7,8 @@
...
+/doc/ @pictographer
```
Comment (sveitser) [inline .github/CODEOWNERS:13]:
```suggestion
```

File: .github/CODEOWNERS [hunk 4]
```diff
@@ -7,8 +7,8 @@
 #    https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
 # for more details.
 
-* alxiong philippecamacho sveitser
-/wallet/ jbearer shenkeyao
-/relayer/ nyospe
-/doc/ pictographer
-/.github/ Ancient123
+* @alxiong @philippecamacho @sveitser
+/wallet/ @jbearer @shenkeyao
```
Comment (sveitser) [inline .github/CODEOWNERS:11]:
```suggestion
/wallet/ @bfish713 @shenkeyao
```

**Comment by sveitser** [comment]
Thanks. Unfortunately the file is also very out of date. I made some suggestions.

**Comment by kevinji** [comment]
Just pushed an updated file.

**Review by sveitser — APPROVED** [review]
LGTM

**Pull request merged**